tests/ValidatorTest.php

```diff
@@ -36,7 +36,7 @@ public function testBadAssocSchemaInput()
         $validator->validate($data, $schema);
     }
 
-    public function testCheck()
+    public function testDeprecatedCheckStillValidates()
```

Comment:
Method name needs clarification - validation in this test is supposed to fail, but the new name falsely implies that success is the intended outcome.

Reply:
How about I rename both to `test.*DelegatesToValidate()`?